src/__tests__/lib/containers/evaluation_queues/EvaluationCard.test.tsx

```diff
@@ -24,7 +25,14 @@ describe('test Evaluation Card', () => {
   let mockOnModifyAccess: jest.Mock
   let mockOnSubmit: jest.Mock
   let mockOnDeleteSuccess: jest.Mock
-  beforeAll(() => server.listen())
+
+  let user: UserEvent
+  beforeAll(() => {
+    server.listen()
+    user = userEvent.setup({
+      pointerEventsCheck: PointerEventsCheckLevel.Never,
```

Comment:
What are you doing here?

Reply:
I believe a change in RTL or userEvent made it so that if a parent element contains `pointer-events: none`, then an error will be thrown. This change removes the check that throws the error.

In the test, `pointer-events: none` is present on the React Bootstrap dropdown. I could not figure out why, and the dropdown options work in Storybook, so I removed the check.